cves/kernel/CVE-2021-43267.yml

```diff
@@ -17,16 +18,16 @@ curated_instructions: |
   Set the version number that you were given in your instructions.
 
   This will enable additional editorial checks on this file to make sure you
-  fill everything out properly. If you are a student, we cannot accept your work
-  as finished unless curated is properly updated.
-curation_level: 0
+  fill everything out properly. If you are a student, we cannot accept your
+  work as finished unless curated is properly updated.
+curation_level: 2
 reported_instructions: |
   What date was the vulnerability reported to the security team? Look at the
   security bulletins and bug reports. It is not necessarily the same day that
   the CVE was created.  Leave blank if no date is given.
 
   Please enter your date in YYYY-MM-DD format.
-reported_date:
+reported_date: '2021-11-04'
```

Comment:
I'm having issue on finding a reference to this reported date. The only date I can find close to this is the NVD creation date of 2021-11-02 and a patch going out on 2021-11-05 in the Fedora 34 Update. Can you confirm this date is correct?

Reply:
Here's where I found it https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2021-43267 but looking at it again this is the report date for the bug, not the CVE. I removed the date and added the bug ID to the list.